cores/esp32/Stream.cpp

```diff
@@ -141,6 +141,114 @@ bool Stream::findUntil(const char *target, size_t targetLen, const char *termina
     return false;
 }
 
+*/
+
+// ******************** ALEXANDRE
+
+ // find returns true if the target string is found
+bool  Stream::find(char *target)
```

Comment:
you should restore the const signatures for these methods.

Reply:
Const signature was not there on original AVR Core code.